### ISSUE TITLE
no edit/

### ISSUE THREAD

**Issue by JDrewes** [owncloud/calendar#347]
After updating from OC 8.2.2 to 8.2.3 and then to 9.0.0, my calendar no longer shows the controls for "edit" and "get link". "share", "download" and "delete" are still there and work as expected. This happens for migrated and new calendars. It is not just missing icons, there is nothing to click there (or I just didn't find it).
Summary: next to the calendar name, there is 3 control icons instead of 5.
![oc9-calendar](https://cloud.githubusercontent.com/assets/17964005/13904784/12cb48fc-eeac-11e5-885f-2327f35c4921.png)
Aside from this, the calendar works well!
### Expected behaviour

5 controls next to the name of each calendar.
### Actual behaviour

3 controls. "edit" and "get link" are missing.
### Server configuration

**Operating system**:
Debian 7

**Web server:**
Apache 2.2.22-13+deb7u6 (latest Debian 7 update)

**PHP version:**
PHP5, 5.4.45-0+deb7u2 (latest Debian 7 update)

**ownCloud version:** (see ownCloud admin page)
ownCloud 9.0.0 (stable)

**Calendar version:** (see ownCloud apps page)
1.0, came with OC9

**Updated from an older ownCloud or fresh install:**
updated from 8.2.2 via 8.2.3

**Signing status (ownCloud 9.0 and above):**
I believe all apps are properly signed, no error messages/warnings.

```
Login as admin user into your ownCloud and access 
http://example.com/index.php/settings/integrity/failed 
paste the results here.
```

No errors have been found.

**List of activated apps:**
- activity: 2.2.1
- calendar: 1.0
- comments: 0.2
- contacts: 1.0.0.0
- dav: 0.1.5
- documents: 0.12.0
- federatedfilesharing: 0.1.0
- federation: 0.0.4
- files: 1.4.4
- files_pdfviewer: 0.8
- files_sharing: 0.9.1
- files_texteditor: 2.1
- files_trashbin: 0.8.0
- files_versions: 1.2.0
- files_videoplayer: 0.9.8
- firstrunwizard: 1.1
- gallery: 14.5.0
- grauphel: 0.6.2
- notes: 2.0.0
- notifications: 0.2.3
- provisioning_api: 0.4.1
- systemtags: 0.2
- tasks: 0.9.0
- templateeditor: 0.1
- updatenotification: 0.1.0

**The content of config/config.php:**

```
Insert your config.php content here

```

<?php
$CONFIG = array (
  'instanceid' => 'something',
  'passwordsalt' => 'something',
  'trusted_domains' => 
  array (
    0 => 'somedomain',
  ),
  'datadirectory' => '/var/www/owncloud/data',
  'overwrite.cli.url' => 'somedomain',
  'dbtype' => 'mysql',
  'version' => '9.0.0.19',
  'dbname' => 'owncloud',
  'dbhost' => 'localhost',
  'dbtableprefix' => 'oc_',
  'dbuser' => 'owncloud',
  'dbpassword' => 'password',
  'installed' => true,
  'mail_from_address' => 'owncloud',
  'mail_smtpmode' => 'php',
  'mail_domain' => 'somedomain',
  'forcessl' => true,
  'theme' => '',
  'defaultapp' => 'calendar',
  'maintenance' => false,
  'preview_libreoffice_path' => '/usr/bin/libreoffice',
  'secret' => 'secret',
  'loglevel' => 0,
  'trashbin_retention_obligation' => 'auto',
  'updatechecker' => false,
  'appstore.experimental.enabled' => true,
);

**Are you using external storage, if yes which one:** local/smb/sftp/...
no
**Are you using encryption:** yes/no
no
**Are you using an external user-backend, if yes which one:** LDAP/ActiveDirectory/Webdav/...
no
### Client configuration

**Browser:**
Firefox 45, Chromium 48.0.2564.116 Ubuntu 15.10 (64-bit)

**Operating system:**
Kubuntu 15.10 with all updates as of March 20 2016

**CalDAV-clients:**
Many, no problems here, works well

**Comment by georgehrke** [comment]
no edit: dup of https://github.com/owncloud/calendar/issues/314
no link: was removed on purpose: hover over the calendar name to see the caldav link

**Comment by JDrewes** [comment]
Okay, but what about the edit option?

**Comment by georgehrke** [comment]
see #314 
That's a bug that will be fixed with Version 1.1
